projects/packages/publicize/src/class-publicize-setup.php

```diff
@@ -42,12 +42,21 @@ public static function on_jetpack_feature_publicize_enabled() {
 		add_action( 'rest_api_init', array( new Connections_Post_Field(), 'register_fields' ), 5 );
 		add_action( 'rest_api_init', array( new REST_Controller(), 'register_rest_routes' ) );
 		add_action( 'current_screen', array( static::class, 'init_sharing_limits' ) );
-		add_action( 'rest_api_init', array( new Jetpack_Social_Settings\Settings(), 'register_settings' ) );
-		add_action( 'admin_init', array( new Jetpack_Social_Settings\Settings(), 'register_settings' ) );
+
+		add_action( 'rest_api_init', array( static::class, 'register_core_options' ) );
```

Comment:
I may be wrong about this, but if we are getting and updating options on simple sites, won't one site be overwriting another site's options since update_option and get_option will run in the context of WPCOM? Perhaps, `update_blog_option` is what we need there. 

Do we even need to register these options on simple sites since we are not showing them in https://github.com/Automattic/jetpack/pull/34665/files?

Reply:
As we tested, `update_option` on wpcom context also runs on the current blog, so it will essentially be an `update_blog_option` for that blog